### PR TITLE
Convert CLI from react-ink to yargs, chalk and Listr.

### LINKED ISSUE #81: Generators: add ability to pass a --force flag to overwrite existing generated files
Right now generators will refuse to do anything if the file they try to generate already exists.

### LINKED ISSUE #93: Generators: When building a sdl file the model is case sensitive
Causing the following failure:

```
➜ yarn rw g sdl userexamples
yarn run v1.21.1
$ /Users/peterp/Personal/redwoodjs/xxx/node_modules/.bin/rw g sdl userexamples
Invoking service generator
Wrote ./api/src/services/userexamples.js 152 bytes
(node:11457) UnhandledPromiseRejectionWarning: No schema definition found for `Userexample`
(node:11457) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
Invoking service generator
Wrote ./api/src/services/userexamples.js 152 bytes
```

`UserExample` exists in schema.prisma

### LINKED ISSUE #96: CLI: Add "pass through" command helper.
A bunch of our commands are aliases to other existing commands:
- build: `yarn rw build <target?>` / `yarn rw build web,api` / `yarn rw build web`
- database: up, save, seed
- tools: lint/ lint fix, open, 

I would love to have a standard way to map a "pass through" command to a command in redwood.  Having a standard way would decrease the amount of code we need to write, and test.

The pass through commands would need to have a way to express usage if used incorrectly, as well as when incorrect arguments are invoked, or on stderr.

It would be great to stream the output from stdout.

### LINKED ISSUE #102: CLI commands that require input are not working
Specifically `yarn rw db save`

### PR DESCRIPTION
**Pull request by peterp**
`react-ink` is slower than I would like it to be so I've converted the CLI to `yargs`. Yargs has a similar interface to the one we created in our original CLI implementation.

You export the following in a file (here's an example of the dev command):
```js
export const command = 'dev [app..]'
export const desc = 'Run development servers.'
export const builder = {
  app: { choices: ['db', 'api', 'web'], default: ['db', 'api', 'web'] },
}
export const handler = ({ app }) => {
   // do stuff...
}
```

And yargs will spit out a nice interface, coerce the args, and run the handler.

Closes #81 
Closes #93 
Closes #96 
Closes #102

### CODE REVIEW

File: packages/cli/src/commands/dbCommands/seed.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+export const command = 'seed'
+export const desc = 'Seed your database with test data.'
+
+export const handler = async () => {
+  const subprocess = execa('node', ['seeds.js'], {
+    shell: true,
+    cwd: getPaths().api.db,
+  })
+  subprocess.stdout.pipe(process.stdout)
+  subprocess.stderr.pipe(process.stderr)
```
Comment (peterp) [inline packages/cli/src/commands/dbCommands/seed.js]: Use `stdio:inherit`

File: packages/cli/src/commands/generate/commands/service.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  )
+
+  return { [outputPath]: template }
+}
+
+export const command = 'service <model>'
+export const desc = 'Generate a service object.'
+export const builder = {
+  crud: { type: 'boolean', default: true },
+  force: { type: 'boolean', default: true },
```
Comment (peterp) [inline packages/cli/src/commands/generate/commands/service.js]: Switch `force` to false.

File: packages/cli/src/commands/generate/helpers.js [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+export const createYargsForComponentGeneration = ({
+  componentName,
+  filesFn,
+}) => {
+  return {
+    command: `${componentName} <name>`,
+    desc: `Generate a ${componentName} component.`,
+    builder: { force: { type: 'boolean', default: false } },
+    handler: async ({ force, ...rest }) => {
+      console.log('yo yo yo')
```
Comment (peterp) [inline packages/cli/src/commands/generate/helpers.js]: ✂️

**Comment by cannikin** [comment]
Want to fix the async output not showing while you're in there? :)

**Comment by peterp** [comment]
@cannikin I think I've fixed it. I underestimated how much love and work you've put into these generators. Thankfully I've just got `scaffold` left! Phew!

**Comment by thedavidprice** [comment]
@peterp Wondering if you’re implementing #81 along the way. Fine either way, but was on my list to dig into next (plenty to do otherwise 😉).

**Comment by peterp** [comment]
@thedavidprice I added `--force`

**Pull request merged**